src/events/guildMemberAdd.ts

```diff
@@ -0,0 +1,15 @@
+import { GuildMember, TextChannel } from 'discord.js';
+
+import { WELCOME } from '../constants';
+
+export const guildMemberAddEvent = async (member: GuildMember) => {
+	const formattedMessage = WELCOME.welcomeMessage
+		.replace(/\{userMention\}/g, `<@${member.id}>`) // @name
+		.replace(/\{userTag\}/g, member.user.tag); // name#1234
+
+	const channel = (await member.guild.channels.get(
+		process.env.WELCOME_CHANNEL,
+	)) as TextChannel;
```

Comment:
This might need to be delayed until https://github.com/typescript-community/community-bot/issues/1 is done, because you could be linking them to a channel they don't have access to yet

Reply:
👍